.circleci/config.yml

```diff
@@ -1,10 +1,9 @@
+
+# This is a stub. CI automation moved to GH actions.
+# Once circleci integration is removed from repo, delete .circleci dir
```

Comment:
Why not removing it in this PR ?

Reply:
Because the integration still exists, and removing the config will just put an ugly ❌ on this check, since a missing config is invalid in circlci world